userguide/virtualmachines.rst

```diff
@@ -889,4 +891,41 @@ Host displays in the |web-ui|. Rancher is now configured and ready for
 use.
 
 For more information on using RancherOS, see the RancherOS
-`Documentation <https://rancher.com/docs/os/v1.x/en/>`__.
+`documentation <https://rancher.com/docs/os/v1.x/en/>`__.
+
+
+.. _Configure Rancher Containers with NFS Pass-through:
+
+Configuring Persistent NFS-Shared Volumes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Rancher supports using a single persistent volume with multiple
+containers. This volume can also be shared with %brand% using NFS.
+%brand% must be configured with specific NFS permissions and a
+`Rancher NFS server
+<https://rancher.com/docs/rancher/v1.6/en/rancher-services/storage-service/rancher-nfs/>`__
+must have a properly configured `stack scoped volume
+<https://rancher.com/docs/rancher/v1.6/en/cattle/volumes/#volume-scopes>`__.
+
+A stack scoped volume is data that is managed by a single Rancher stack.
+The volume is shared by all services that reference it in the stack.
+
+Configure NFS sharing for a stack scoped volume by setting specific
+options in the command line of the Rancher NFS server and the %brand%
+system:
+
+* Log in to the Rancher NFS server and modify :file:`/etc/exports`. Add
+  an entry for the NFS shared directory, typically :file:`/nfs`, with
+  several permissions options:
+  :samp:`/nfs	{IP}(rw,sync,no_root_squash,no_subtree_check)`.
+  :literal:`{IP}` is the IP address of the client and can also be set
```

Comment:
Do curly brackets in :literal: markup work the same as in :samp:?  If not, this can be marked up correctly as: `` *IP* is the IP address... ``